### ISSUE TITLE
jQuery slim build reports "invalid version"

### ISSUE THREAD

**Issue by topaxi** [GoogleChrome/lighthouse#3624]
I'm using the jQuery slim build on a website, lighthouse is reporting "invalid version", my guess is it parses some kind of semver like versions while the jQuery slim version looks like this:
```
3.2.1 -ajax,-ajax/jsonp,-ajax/load,-ajax/parseXML,-ajax/script,-ajax/var/location,-ajax/var/nonce,-ajax/var/rquery,-ajax/xhr,-manipulation/_evalUrl,-event/ajax,-effects,-effects/Tween,-effects/animatedSelector
```

**Comment by patrickhulce** [comment]
Thanks for the report! Could you clarify what you mean by Lighthouse is reporting "invalid version"? Is there a specific audit that is failing or a fatal error? A copy of the report/json here or screenshot with test URL would be helpful here too if you have them.

**Comment by topaxi** [comment]
Under "Best Practices" - "Includes front-end JavaScript libraries with known security vulnerabilities":

![best-practices](https://user-images.githubusercontent.com/213788/31831948-7c7dcdf6-b5c5-11e7-860d-0ed72f42abc4.png)

**Comment by patrickhulce** [comment]
Great thanks for sharing the URL @topaxi! I can reproduce.

**Comment by patrickhulce** [comment]
Indeed our bug, basically need to wrap https://github.com/GoogleChrome/lighthouse/blob/76e51f3821dad783763873e860e486949146d402/lighthouse-core/audits/dobetterweb/no-vulnerable-libraries.js#L69 in try/catch

**Comment by paulirish** [comment]
we have two fixes here:

1. apply a trycach so we dont fail so hard (as patrick's comment points out)
2. js-library-detector lib correctly extracts the version in this case.

@topaxi can you report this upstream to js-library-detector?  they should be trimming off anything separated by whitespace, IMO.

**Comment by johnmichel** [comment]
@paulirish @topaxi This should be resolved in `v4.3.0`, which was just published to both `npm` and the Chrome Web Store.  Please let me know if you see any other strange behavior.
